hapi-base/src/main/java/ca/uhn/hl7v2/parser/FixFieldDataType.java

```diff
@@ -137,7 +144,9 @@ public static void fixOBX5(Segment segment, ModelClassFactory factory, ParserCon
             invalidOBX2Type = System.getProperty(INVALID_OBX2_TYPE_PROP);
         }
 
-        fix(segment, 2, 5, defaultOBX2Type, invalidOBX2Type, factory, parserConfiguration);
+        if (ENABLE_DEFAULT_OBX2_DEFAULT_VALUE){

```

Comment:
It is hard for me to tell if this change is behaving as intended because there is no test exercising it.

Reply:
Updated. Thanks.